### DISCUSSION TITLE
Language change after indexing

### DISCUSSION THREAD

**Discussion by benevenuti15** [sepinf-inc/IPED#1120] [Q&A] [answered]
Good afternoon everyone.
I would like to ask a question regarding the language setting. It is possible to change the language after indexing or it is only possible to do it before indexing, directly in the "LocalConfig.txt" file

**Comment by lfcnassif** [comment, marked as answer]
Hi, it will be possible in the future 4.0.0 release.

**Reply by benevenuti15** [reply]
It's great! thanks
